src/lib/fees/model.ts

```diff
@@ -32,23 +34,12 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld }) => {
   const midpointUsd = bn(FEE_CURVE_MIDPOINT_USD)
   const feeCurveSteepness = bn(FEE_CURVE_STEEPNESS_K)
 
-  const foxDiscountPercent = BigNumber.minimum(
+  // the following raw values are before the realities of integer bps on-chain
+  const foxDiscountPercentRaw = BigNumber.minimum(
     bn(100),
     foxHeld.times(100).div(bn(FEE_CURVE_FOX_MAX_DISCOUNT_THRESHOLD)),
   )
-
-  if (tradeAmountUsd.lt(noFeeThresholdUsd)) {
-    return {
-      feeBps: bn(0),
-      feeUsd: bn(0),
-      feeUsdDiscount: bn(0),
-      foxDiscountPercent,
-      feeUsdBeforeDiscount: bn(0),
-      feeBpsBeforeDiscount: bn(0),
-    }
-  }
-
-  const feeBpsBeforeDiscount = minFeeBps.plus(
+  const feeBpsBeforeDiscountRaw = minFeeBps.plus(
```

Comment:
Ditto terminology, not clear what this does unless you read the whole PR

Reply:
actioned in 19be7d23c47677edafae86e2457a7bc278d0e430